Services/PersonalDesktop/ItemsBlock/classes/class.ilPDSelectedItemsBlockGUI.php

```diff
@@ -778,7 +778,7 @@ public function confirmedUnsubscribe()
 					
 					default:
 						// do nothing
-						continue;
+						continue 2;
 				}											
 		
```

Comment:
@mjansenDatabay : Is it necessary to execute the following forum lines or is a "continue 2" correct here.

Reply:
The code MUST only be executed for courses and groups, NOT for the default. So a `continue 2;` should be okay here.